src/Sentry/SentryOptions.cs

```diff
@@ -1324,6 +1325,18 @@ public SentryOptions()
         );
 
         NetworkStatusListener = new PollingNetworkStatusListener(this);
+
+#if NET8_0_OR_GREATER
```

Comment:
remove preprocessor directive

Consuming projects could use a newer SDK (currently we require 6.0.400 for the source generator to work) targeting and older TFM, and the Source Generator would still work.

Reply:
Looks like from https://github.com/getsentry/sentry-dotnet/pull/4333 we need to leave the `#if` clause in there but we can change it to `#if NET5_0_OR_GREATER`